packages/client/tests/functional/raw-queries/send-type-hints/tests.ts

```diff
@@ -6,9 +7,10 @@ declare let prisma: PrismaClient
 declare let Prisma: typeof PrismaNamespace
 
 testMatrix.setupTestSuite(
-  (suiteConfig) => {
-    test('Buffer ($queryRaw)', async () => {
-      if (suiteConfig.provider === 'mysql') {
+  ({ provider, providerFlavor }) => {
+    // TODO the buffer returned by the driver adapter seems to be an ArrayBufferView, not a Node.js Buffer
```

Comment:
Now this is weird as it has to go all the way through our code and the buffers are theoretically always constructed by us. I would expect this test to fail for other reasons (either some conversion error or wrong data like byte values of characters of `{}` string in the response) but not like this.